blueprints/app/files/app/app.js

```diff
@@ -3,14 +3,12 @@ import Resolver from 'ember-resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
 
-var App;
-
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
-App = Ember.Application.extend({
+const App = Ember.Application.extend({
```

Comment:
definition was moved above by a PR, someone's linter complained #3793. I'm for putting it back to one line, just letting you know reasoning.